soc/v2/arm/fvp_aemv8a/Kconfig

```diff
@@ -2,8 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config SOC_SERIES_FVP_AEMV8A
-	bool "ARM FVP AEMv8A AArch64 Series"
 	select ARM64
 	select SOC_FAMILY_ARM64
```

Comment:
the SOC_FAMILY_ARM64 is not defined anywhere.

Are we missing a Kconfig.soc at this level: https://github.com/zephyrproject-rtos/zephyr/tree/80451dd4eb46233ef2c81c98f48195c650573931/soc/v2/arm ?

And if yes, should we move the soc.yml one level up, and describe the soc family in the soc.yml in addition.

Reply:
Seems mps3 did not have this done properly, has been fixed